src/connect-cloud-health/ConnectHealth.d.ts

```diff
@@ -1,5 +1,6 @@
-import { HealthChecker, ReadinessCheck, LivenessCheck, ShutdownCheck } from '@cloudnative/health';
+import { HealthChecker, StartupCheck, ReadinessCheck, LivenessCheck, ShutdownCheck, PingCheck } from '@cloudnative/health';
```

Comment:
ping check is not mentioned in the README.md, is this something we should add in?

Reply:
We probably should document it, but for the moment I'll leave it as an "Easter egg" ;-)